community/index.html

```diff
@@ -103,92 +103,30 @@ <h1>Community</h1>
 <!-- Support section -->
 <div class="container flex info">
   <div class="community">
-    <div class="row">
-  <div class="col-sm-3">
-    <h2><span id="support"></span>Support</h2>
-  </div>
-  <div class="col-sm-9">
-    <div class="row">
-      <div class="col-sm-6">
-        <h3>Gitter Chat</h3>
-        <p><a href="https://gitter.im/Knotx/Lobby">Gitter Chat</a> is a way for users to chat
-          with the Knot.x team. Feel free to leave a message, even if we’re not around, we will
-          definitely respond to you when available.</p>
-      </div>
-      <div class="col-sm-6">
-        <h3>User Group</h3>
-        <p><a href="https://groups.google.com/forum/#!forum/knotx">Knot.x User Group</a> is
-          the one of the way to ask questions and communicate with the Knot.x
-          team and other users.</p>
-      </div>
-    </div>
-    <div class="row">
-      <div class="col-sm-6">
-        <h3>Contributors Group</h3>
-        <p><a href="https://groups.google.com/forum/#!forum/knotx-contributors">Knot.x
-          Contributors Group</a> is the one of the way to ask questions and communicate with the Knot.x
-          team and other users.</p>
-      </div>
-      <div class="col-sm-6">
-        <h3>Github Issues</h3>
-        <p>All feature requests and bugs can be filed as issues on
-          <a href="https://github.com/Knotx/knotx/issues">Github</a>.
-          Do not use Github issues to ask questions, post them on the User Group.</p>
-      </div>
-    </div>
-  </div>
-</div>
-<hr>
-<div class="row">
-  <div class="col-sm-3">
-    <h2><span id="news"></span>Updates</h2>
-  </div>
-  <div class="col-sm-9">
-    <p>Knot.x <a href="/blog">blog page</a> is the place you can find such information like
-      tutorials and articles. We recommend also to follow our <a href="https://twitter.com/knotx_project">@knotx_project</a>
-      twitter account if you are interested in current news / updates.
-    </p>
-  </div>
-</div>
-<hr>
-<div class="row">
-  <div class="col-sm-3">
-    <h2><span id="contributing"></span>Contributing</h2>
-  </div>
-  <div class="col-sm-9">
-    <p>Bug fixes and feature contributions in the form of pull requests are welcome. If you have a
-      non-trivial change in mind that you'd like to discuss, feel free to contact us on
-      <a href="https://gitter.im/Knotx/Lobby">Gitter</a> or the
-      <a href="https://groups.google.com/forum/#!forum/knotx">user group</a>.
-    </p>
-  </div>
-</div>
-    <hr>
-    <!-- Core Team -->
     <div class="row">
       <div class="col-sm-3">
-        <h2><span id="core-team"></span>Core Team</h2>
+        <h2><span id="core-team"></span>Commiters</h2>
```

Comment:
```suggestion
        <h2><span id="core-team"></span>Core developers</h2>
```
This will correspond better with the section below.